data/human/culture conversations.txt

```diff
@@ -120,6 +149,38 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Forest Fire"
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting, rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`
+			choice
+				`	"Hey! What's happening? I can help!" (Shout.)`
+					goto help
+				`	(Look for someone not in motion.)`
+			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or one some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`
+			choice
+				`	"I have a ship out there. Let me help."`
+			`	Her eyes widen and she looks around, sharply catching the attention of one of the many people hurrying about.`
+				goto fight
+			label help
+			`	A young man who looks suspiciously like he might've been air traffic control twenty minutes ago glances at you and skids to a stop. "Oh, you're a pilot! Can your ship help?"`
+			choice
+				`	"Yes."`
+			label fight
+			`	The dock-hands scrambled to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely remembered what every button did before the Air Traffic Controller cleared you to depart, navigating your craft towards the fire.`
+			`	The ragging fire flared like an inferno, throwing up black smoke as you approached. Animals scrambled towards safer grounds as you dropped the retardants, making sure to keep your speed low before going back for another batch.`
```

Comment:
Seems odd to switch to past tense all of a sudden. The time skip can either happen between paragraphs, or you can end this sequence with a dialog option a là "keep dropping retardant" to make it more noticeable.

Reply:
Proposal:
```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, navigating your craft towards the fire.`
			`	The raging fire flares like an inferno, throwing up black smoke in deep billowing waves as you approach. As you drop the retardants, making sure to keep your speed low, animals scramble towards safer grounds. You try to make sure you've not emptied right on top of them each time as you go back for another batch.`
```

Reply:
lgtm